src/activities/jumbled_words/jumbled_words.js

```diff
@@ -0,0 +1,177 @@
+/* GCompris - jumbled_words.js
+ *
+ * Copyright (C) 2016  Komal Parmaar <parmaark@gmail.com>
+ *
+ * Authors:
+ *    Komal Parmaar <parmaark@gmail.com>
+ *
+ *   This program is free software; you can redistribute it and/or modify
+ *   it under the terms of the GNU General Public License as published by
+ *   the Free Software Foundation; either version 3 of the License, or
+ *   (at your option) any later version.
+ *
+ *   This program is distributed in the hope that it will be useful,
+ *   but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *   MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *   GNU General Public License for more details.
+ *
+ *   You should have received a copy of the GNU General Public License
+ *   along with this program; if not, see <http://www.gnu.org/licenses/>.
+ */
+
+.pragma library
+.import QtQuick 2.0 as Quick
+.import GCompris 1.0 as GCompris
+.import QtGraphicalEffects 1.0 as QtGraphicalEffects
+
+.import "qrc:/gcompris/src/core/core.js" as Core
+
+var url = "qrc:/gcompris/src/activities/jumbled_words/resource/"
+var url1 = "qrc:/gcompris/src/activities/lang/resource/words_sample.json"
+var defaultLevelsFile = "qrc:/gcompris/src/activities/lang/resource/words_sample.json";
+
+var currentLevel
+var maxLevel
+var maxSubLevel
+var currentSubLevel
+var items
+var levels;
+var level;
+var temp;
```

Comment:
Not a good name for a variable.
For all the variables, are you sure they need to be global?